bundle/compliance/cis_eks/rules/cis_3_2_11/rule.rego

```diff
@@ -18,12 +18,4 @@ rule_evaluation {
 	audit.not_process_contains_variable("--feature-gates", "RotateKubeletServerCertificate", ["featureGates", "RotateKubeletServerCertificate"])
 }
 
-rule_evaluation {
-	audit.process_contains_key_with_value("--rotate-server-certificates", "true")
-}
-
-rule_evaluation {
-	audit.get_from_config(["serverTLSBootstrap"])
-}
-
```

Comment:
why did you remove this?

Reply:
There is no notion of `serverTLSBootstrap` and `rotate-server-certificates` in the rule's metadata, so these variables are irrelevant and as far as I understand were probably a wrong carry-on from `k8s rule 4.2.12`.

In addition to that, there is a notion of `rotate-kubelet-server-certificate` but it is inside the remediation, since we are not doing any remediations, I've assumed that we shouldn't validate that.

@ofiriro3 please correct me if I'm wrong in any of the above statements.

Reply:
Hi,

I went over the rule again and I based the definition of the rule on the remediation suggested by the benchmark.

I'll try to elaborate:

> Remediation Method 1:
If modifying the Kubelet config file, edit the kubelet-config.json file /etc/kubernetes/kubelet/kubelet-config.json and set the below parameter to true

If it's within the configuration file, the parameter should be named `"RotateKubeletServerCertificate":true`

> Remediation Method 2:
If using a Kubelet config file, edit the file to set RotateKubeletServerCertificate to
true.
If using executable arguments, edit the kubelet service file
/etc/systemd/system/kubelet.service.d/10-kubelet-args.conf on each worker node
and add the below parameter at the end of the KUBELET_ARGS variable string.

Based on that, we can infer that there is a process argument named `--rotate-kubelet-server-certificate`, or a least we can define it that way.

3. I remember it a bit vaguely, but in order for kubelet to use the certificate while communicating, it needs to have serverTLSBootstrap = true.

That being said, its is not very explicit (at least in the title so maybe I am wrong here)

@jeniawhite @oren-zohar